docs/source/howto/data.rst

```diff
@@ -260,6 +260,359 @@ Finding data
 Organizing data
 ===============
 
+Grouping sets of nodes together
+-------------------------------
+
+AiiDA's database is great for automatically storing all your data, but sometimes it can be tricky to navigate this flat data store.
+To create some order in this mass of data, you can *group* sets of nodes together, just as you would with files in folders on your filesystem.
+A folder, in this analogy, is represented by the :py:class:`aiida.orm.groups.group.Group` class.
```

Comment:
The reference appears to be broken, also I'd suggest to shorten the class names etc:
```suggestion
A folder, in this analogy, is represented by the :py:class:`~aiida.orm.groups.group.Group` class.
```